tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -33,3 +33,25 @@ def test__contains__(self):
         assert not isaspect(str)
         exc.match("<class 'str'> is not an "
                   'aspectclass or an instance of an aspectclass')
+
+    def test_get(self):
+        list_of_aspect = AspectList(
+            [Metadata.CommitMessage.Shortlog, Metadata.CommitMessage.Body])
+        assert list_of_aspect.get(Metadata.CommitMessage.Shortlog) is \
```

Comment:
I can't change it to unittest style assert because the pytest fixture will not injected even after adding `@pytest.mark.usefixtures`